go.mod

```diff
@@ -38,7 +30,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
 	github.com/hashicorp/consul-awsauth v0.0.0-20220713182709-05ac1c5c2706
 	github.com/hashicorp/consul-net-rpc v0.0.0-20221205195236-156cfab66a69
-	github.com/hashicorp/consul/api v1.24.0
+	github.com/hashicorp/consul/api v1.25.0
```

Comment:
Was this supposed to be a larger version bump?

Reply:
yes, there's a some methods added to the public API that would make it a minor version bump, the other versions off 1.15.x and 1.14.x probably should've been new minor versions as well but we didn't have the version numbers available there